src/algebra/group_power/basic.lean

```diff
@@ -548,6 +548,22 @@ variable [ordered_semiring R]
 | 0     := zero_le_one
 | (n+1) := mul_nonneg H (pow_nonneg _)
 
+theorem pow_add_le {x y : R} {n : ℕ} (hx : 0 ≤ x) (hy : 0 ≤ y) (hn : n ≠ 0) :
```

Comment:
```suggestion
theorem pow_add_pow_le {x y : R} {n : ℕ} (hx : 0 ≤ x) (hy : 0 ≤ y) (hn : n ≠ 0) :
```
otherwise I would assume this is a statement `x ^ (n + m) ≤ _`